Network/Network.h

```diff
@@ -54,6 +54,10 @@ namespace WPEFramework {
             Network(const Network&) = delete;
             Network& operator=(const Network&) = delete;
 
+            //Private variables
+            bool isNetworkPluginInitialized;
```

Comment:
Can you rename the variable with "m_" prepend..
Something like "m_isPluginReady"